sdf.md

```diff
@@ -137,15 +136,14 @@ The odmObject keyword denotes zero or more Object definitions. A object may cont
 
 - Qualities of odmObject
 
-| Quality | Type | Optional | Description |
-|---|---|---|---|
-|id| integer | yes | internal unique identifier for the definition |
-|name|string|yes|human readable name|
-|description|string|yes|human readable description|
-|title|string|yes|human readable title to display|
-|optional| boolean|yes|defines whether this element is optional in an implementation|
-|include|array|yes|reference to definitions to be included|
-| odmType|object|yes|reference to a definition to be used as a template for a new definition|
+| Quality | Type | Required | Description | Default |
+|---|---|---|---|---|
+|id| integer | yes | internal unique identifier for the definition |  N/A |
```

Comment:
This makes ID required. Which I think makes sense, but is not a simple polarity change as for the others.